test/Tester/RelationalUtilities/RelationalStorageForTesting.cs

```diff
@@ -17,11 +17,11 @@ public abstract class RelationalStorageForTesting
                 {AdoNetInvariants.InvariantNameMySql, cs => new MySqlStorageForTesting(cs)}
             };
 
-        private readonly IRelationalStorage storage;
```

Comment:
I'll look into this again, if I still have another idea of implementation I'll post an example.